andela/ah-aquaman-frontend
# #164069178 Users Signup

### What does this PR do?
- This PR creates a signup page used by users to register with Authors Haven 

### Description of Task to be completed?
- Provide the user with a signup form
- Provide descriptive error messages
- Reroute the user to the login page on email verification

### How should this be manually tested?
- Clone the [repo](https://github.com/andela/ah-aquaman-frontend) and cd into ah-aquaman-frontend
- checkout the branch `ft-users-should-signup-164069178`
- Run the commands below to install packages and run the server
          - npm install
          - npm start
- Navigate to `/signup` 
- Provide `username`, `email` and `password` to register
- This feature is hosted [here](https://ah-aquaman-frontend-stag-pr-12.herokuapp.com/signup)

### What are the relevant pivotal tracker stories?

- [#164069178](https://www.pivotaltracker.com/story/show/164069178)

### Screenshots
<img width="1423" alt="Screenshot 2019-04-11 at 07 57 50" src="https://user-images.githubusercontent.com/6951438/55931808-9edcce80-5c2f-11e9-8525-49d2c971da57.png">
